src/frontend/include/yaksi.h

```diff
@@ -71,6 +71,11 @@ struct yaksi_request_s;
 #define YAKSI_REQUEST_GET_OBJECT_ID(handle) \
     ((handle) << (64 - YAKSI_REQUEST_OBJECT_ID_BITS) >> (64 - YAKSI_REQUEST_OBJECT_ID_BITS))
 
+typedef enum {
+    YAKSI_REQUEST_NONBLOCKING,
+    YAKSI_REQUEST_BLOCKING,
+} yaksi_request_nb_kind_e;
```

Comment:
It is enum of boolean values. I found the name yaksi_request_*nb*_kind_e particularly not helping. Why not just use `bool is_blocking` in the interface rather than the indirect enum type?

Reply:
addressed in the latest push